src/css/custom.css

```
@@ -330,3 +330,18 @@ html:not([data-theme='dark']) .DocSearch-Sidepanel-Prompt--submit {
+   since every TOC entry belongs to the (hidden) manual guide. The TOC column
+   width is kept, so toggling panes does not reflow the content. */
+html[data-setup-pane='notoc'] .theme-doc-toc-desktop,
+html[data-setup-pane='notoc'] .theme-doc-toc-mobile {
+  display: none;
+}
+
+/* Wrap long single-line commands instead of scrolling horizontally */
+.wrap-code pre,
+.wrap-code code {
+  white-space: pre-wrap !important;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)